tests/conftest.py

```diff
@@ -2481,6 +2482,16 @@ def sequencing_platform() -> str:
     return SequencingPlatform.ILLUMINA
 
 
+# Nallo fixtures
+@pytest.fixture(scope="function", name="nallo_context")
+def fixture_nallo_context(
```

Comment:
Adding an alias to a fixture breaks the PyCharm "find usages" functionality
```suggestion
@pytest.fixture(scope="function")
def nallo_context(
```